### PR TITLE
Release 1.0.0

### LINKED ISSUE #105: pypi.org release 1.0
publish the 1.0 hive release on pypi.org. [documentation](https://packaging.python.org/en/latest/)

Depends on #106

### LINKED ISSUE #106: create release v1.0.0
release 1.0.0 includes all changes reflected by open-source licensing and publishing of hive. this should replace the 0.x series.

### PR DESCRIPTION
**Pull request by nreinicke**
First open source release.

Closes #105 
Closes #106

### CODE REVIEW

**Comment by nreinicke** [comment]
@rjf-nrel @jhoshiko - I've uploaded the package to PyPI: https://pypi.org/project/nrel.hive/

Can you confirm that you're able to install from that source?

```bash
pip install nrel.hive
```

@jhoshiko - I've added you as a project owner on pypi.org
@rjf-nrel - do you have a pypi.org handle that I can use to add you as an owner to the project?

**Comment by robfitzgerald** [comment]
> do you have a pypi.org handle that I can use to add you as an owner to the project?

rjf-nrel

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# HIVE™ :honeybee:
```
Comment (robfitzgerald) [inline README.md:1]: removing the colon breaks the emoji here

**Review by robfitzgerald — CHANGES_REQUESTED** [review]
runs for me. one small typo fix. thanks! 🐝

**Review by robfitzgerald — APPROVED** [review]
:honeybee:

**Comment by jhoshiko** [comment]
Looks like everything is working on my end too. Thanks Nick! 🐝

**Review by jhoshiko — APPROVED** [review]

**Pull request merged**